packages/app-builder-lib/src/util/yarn.ts

```diff
@@ -93,26 +97,21 @@ async function installDependencies(config: Configuration, appDir: string, option
   const additionalArgs = options.additionalArgs
 
   log.info({ platform, arch, appDir }, `installing production dependencies`)
-  let execPath = process.env.npm_execpath || process.env.NPM_CLI_JS
```

Comment:
Shouldn't we allow these to be still an override to `getPackageToolPath`? I think that would allow us to handle `npx electron-builder` correctly, right? Not sure if we have a unit test for that (or if we even should have one tbh). Curious on your thoughts

Reply:
Even when using 'npx electron-builder', there should still be a PM (Package Manager, which currently defaults to npm), so it should work fine. I haven't tried using 'npx electron-builder' directly - currently there are no corresponding unit tests for this scenario, and we should add such unit tests.